layouts/_default/list.html

```diff
@@ -16,6 +16,15 @@
         <h2 class="post__title">
           <a href="{{.RelPermalink}}">{{ .Title }}</a>
         </h2>
+        {{if .Site.Params.showSummary }}
+        {{ .Summary }}
+        {{if .Truncated }}
```

Comment:
slight formatting issue: this should be indented as a nested `if` block. Not sure about this as @jakewies set up `prettier`. Any thoughts?

Reply:
If I run ```yarn format``` i get the below changes?
![image](https://user-images.githubusercontent.com/4903148/87456025-faacb900-c623-11ea-992f-d1a299328f25.png)

Is it okay to push?

Reply:
I had another comment regarding formatting in an earlier review but I have closed it in favor of this one, for simplicity.

I wouldn't want you to push those changes. `yarn format` will adjust files that are outside the scope of this PR. We have a git hook that runs before each commit. The hook will format any staged files for commit.

That's why I'm a bit confused as to how the `layouts/_default/list.html` made it to this PR without being formatted. 

Are you sure that you have followed the `CONTRIBUTING.md` file instructions for setting up your development workflow?

What OS are you on?

What text editor are you using?